tests/asgi/test_asgi_servers.py

```diff
@@ -109,17 +109,20 @@ def _run_server_isolated(process_factory, host, port):
         server.send_signal(signal.CTRL_C_EVENT)
         try:
             server.wait(timeout=10)
-        except (KeyboardInterrupt, subprocess.TimeoutExpired):
+        except KeyboardInterrupt
             pass
```

Comment:
I wasn't sure about this case. When KeyboardInterrupt is raised, does that mean the process got the CTRL+C and exited cleanly?

Reply:
It's a bit strange. I think there are bugs in how python handles this.
I think that this ctrl_c signal is sent to the whole process tree, not only the process you send it to, so the caller recieves a ctrl_c event, and that does not make any sense